test/e2e/tests/multichain/asset-list.spec.ts

```diff
@@ -0,0 +1,192 @@
+        await sendPage.check_pageIsLoaded();
+        await sendPage.fillRecipient(
+          '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+        );
+        await sendPage.clickAssetPickerButton();
+        const assetPickerItems = await sendPage.getAssetPickerItems();
+        assert.equal(
+          assetPickerItems.length,
+          1,
+          'Only one asset should be shown in the asset picker',
+        );
+      },
+    );
+  });
+  it('switches networks when clicking on swap for a token on another network', async function () {
+    await withFixtures(
+      buildFixtures(this.test?.fullTitle() as string),
+      async ({
+        driver,
+        ganacheServer,
+      }: {
+        driver: Driver;
+        ganacheServer?: Ganache;
+      }) => {
+        await loginWithBalanceValidation(driver, ganacheServer);
+        const assetListPage = new AssetListPage(driver);
+        await assetListPage.waitUntilAssetListHasItems(4);
+        await assetListPage.clickOnAsset('LineaETH');
+        await driver.clickElement('[data-testid="token-overview-button-swap"]');
+        const toastTextElement = await driver.findElement('.toast-text');
+        const toastText = await toastTextElement.getText();
+        assert.equal(
+          toastText,
+          "You're now using Linea Sepolia",
+          'Toast text is correct',
+        );
+      },
+    );
+  });
+  it('shows correct asset and balance when swapping on a different chain', async function () {
+    await withFixtures(
+      buildFixtures(this.test?.fullTitle() as string),
+      async ({
+        driver,
+        ganacheServer,
+      }: {
+        driver: Driver;
+        ganacheServer?: Ganache;
+      }) => {
+        await loginWithBalanceValidation(driver, ganacheServer);
+        const headerNavbar = new HeaderNavbar(driver);
+        const assetListPage = new AssetListPage(driver);
+        const selectNetworkDialog = new SelectNetwork(driver);
+        await headerNavbar.clickSwitchNetworkDropDown();
+        await selectNetworkDialog.selectNetworkName(LINEA_NAME_MAINNET);
+        await assetListPage.waitUntilAssetListHasItems(2);
+
+        await assetListPage.clickOnAsset('Ethereum');
+
+        const swapButton = await driver.findElement(
+          '[data-testid="token-overview-button-swap"]',
+        );
+        await swapButton.click();
+        const toastTextElement = await driver.findElement('.toast-text');
+        const toastText = await toastTextElement.getText();
+        assert.equal(
+          toastText,
+          `You're now using ${LOCALHOST}`,
+          'Toast text is correct',
+        );
+        const balanceMessageElement = await driver.findElement(
+          '.prepare-swap-page__balance-message',
+        );
+        const balanceMessage = await balanceMessageElement.getText();
+        assert.equal(
+          balanceMessage.replace('Max', '').trim(),
+          'Balance: 24.9956',
+          'Balance message is correct',
```

Comment:
nit: we're using the const `EXPECTED_CURRENT_NETWORK_BALANCE_USD` elsewhere in the test rather than hardcoding the string. Does that make sense here too?